src/components/LoginForm/LoginForm.jsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Form, Field } from "react-final-form";
+import { useNavigate, Link } from "react-router-dom";
+import "./LoginForm.scss";
+import { useDispatch } from "react-redux";
+import { loginUserWithLocalStorage } from "../../actions/authActions";
+
+const LoginForm = ({ errorMessage, setErrorMessage }) => {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const validate = (values) => {
+    const errors = {};
+    if (!values.userName) {
+      errors.userName = "*";
+    }
+    if (!values.password) {
+      errors.password = "*";
+    }
+    return errors;
+  };
```

Comment:
could you please explain?

Reply:
vaiya It's from React Final Form Validation. When I click login button without giving any value, it shows the asterisk sign. Should I remove this validation?